content/docs/reference/config/installation-options/index.md

```diff
@@ -137,6 +137,7 @@ To customize Istio install using Helm, use the `--set <key>=<value>` option in H
 | --- | --- | --- |
 | `global.hub` | `gcr.io/istio-release` | `Default hub for Istio images. Releases are published to docker hub under 'istio' project. Daily builds from prow are on gcr.io, and nightly builds from circle on docker.io/istionightly` |
 | `global.tag` | `release-1.1-latest-daily` | `Default tag for Istio images.` |
+| `global.imagePullSecrets` | `` | `Lists the secrets you need to use to pull Istio images from a secure registry.` |
```

Comment:
this is the wrong way to document this option.  Please modify values.yaml for the relevant component as this documentation is autogenerated.